utils/fixed_decimal/src/compact.rs

```diff
@@ -19,13 +19,13 @@ use crate::ParseError;
 /// nor a sign in the exponent, and behaves differently in pluralization.
 #[derive(Debug, Clone, PartialEq)]
 pub struct CompactDecimal {
-    significand: FixedDecimal,
+    significand: SignedFixedDecimal,
```

Comment:
In this PR, we are only focusing on the composition, and all the current uses of `FixedDecimal` can be mapped to `SignedFixedDecimal` because `FixedDecimal` was signed.

In the next PRs, we can see if some places in the code can use `UnsignedFixedDecimal` instead of signed.

Reply:
In `ScientificDecimal` you replaced `FixedDecimal` with `UnsignedFixedDecimal` though

Reply:
done

Reply:
We should eventually make `Compact` composable, so that you can have an `UnsignedCompactDecimal`, for example. We can do that later in a different PR.